CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
 
 ### Fixes
 
+* Fixes an issue where each element was being assigned to a separate chunk in chunk_by_title, by dropping the coordinates metadata field
```

Comment:
Please provide a bit more context, which may mean longer bullets than traditionally has been the case:

What was the problem? What was fixed? Why does it matter?

Reply:
Addressed with https://github.com/Unstructured-IO/unstructured/pull/1374/commits/0f6b0dcb6a258b867c5f904515b9ca8e6c70bd5d